packages/babel-preset-gatsby/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "babel-preset-gatsby",
+  "version": "2.5.0",
```

Comment:
Not really sure what to use here. I copied from `babel-plugin-remove-graphql-queries`. Maybe we want to add a changelog as well?

Reply:
Nope! Don't add a changelog, it'll get autogenerated by Lerna I believe.

cc @pieh sanity check me on this one :)

Reply:
let's start with 1.0.0 (or lower heh) - all our packages have independent versions